src/sentry/runner/initializer.py

```diff
@@ -47,6 +49,23 @@ def init_plugin(plugin):
         for cls in plugin.get_custom_contexts() or ():
             contexttype(cls)
 
+    if (hasattr(plugin, 'get_cron_schedule') and plugin.is_enabled()):
+        settings.CELERYBEAT_SCHEDULE.update(plugin.get_cron_schedule())
+
+    if (hasattr(plugin, 'get_worker_imports') and plugin.is_enabled()):
+        settings.CELERY_IMPORTS += (plugin.get_worker_imports(),)
```

Comment:
Won't this just be `settings.CELERY_IMPORTS += plugin.get_worker_imports()`? Otherwise, you'll be appending a tuple, to a tuple.

And also might wanna coerce `get_worker_imports` to a tuple as well since it'll error if the types are mismatched.

So:

```python
imports = plugin.get_worker_imports()
if imports:
    settings.CELERY_IMPORTS += tuple(imports)
```

This will also handle the case where it may return `None` so it doesn't blow up there.